Source/Core/DolphinQt/NetPlay/NetPlayDialog.cpp

```diff
@@ -98,6 +99,8 @@ void NetPlayDialog::CreateMainLayout()
   m_main_layout = new QGridLayout;
   m_game_button = new QPushButton;
   m_start_button = new QPushButton(tr("Start"));
+  m_auto_buffer_box = new QCheckBox;
+  m_auto_buffer_label = new QLabel(tr("Auto Buffer:"));
```

Comment:
Let's get rid of this separate label and use the QCheckBox's native label instead, like the rest of the UI code. :)

```c++
m_auto_buffer_box = new QCheckBox(tr("Auto Buffer"));
```

Also not sure whether I prefer the checkbox to the left or right side of the spinbox.